### PR TITLE
Extract and plot model coefficients

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Extract and plot model coefficients

In this PR, coefficients per CellProfiler feature are extracted and then plotted per class (Healthy or Failing cell type).

### CODE REVIEW

File: 5.machine_learning/figures/logistic_regression_final_healthy_coefficients_plot.png
Comment (MikeLippincott) [inline 5.machine_learning/figures/logistic_regression_final_healthy_coefficients_plot.png:1]: Consider adding a shuffled model too? Consider making text larger

Comment (MikeLippincott) [reply 5.machine_learning/figures/logistic_regression_final_healthy_coefficients_plot.png:1]: I would also consider making the squares slightly larger

**Review by MikeLippincott — APPROVED** [review]
LGTM!

File: 5.machine_learning/figures/logistic_regression_final_failing_coefficients_plot.png
[the remark this answers is not in the document]
Comment (jenna-tomkinson) [reply 5.machine_learning/figures/logistic_regression_final_failing_coefficients_plot.png:1]: I increased the size of the font, and I made the width a bit smaller for the plot. I also confirmed that the legend is correctly matching the failing plot file name. Thanks for the suggestion!

**Comment by jenna-tomkinson** [comment]
@MikeLippincott Thank you for the review! I will be merging now!

**Pull request merged**